### PR TITLE
Fix #293837: Plugin: WIP: Add ability to compare sameness of MuseScore QML elements.

### PR DESCRIPTION
**Pull request by DLLarson**
This PR is a work in progress to test ideas for comparing objects in QML scripts.

Note that there may be several patches using different solutions. When the PR is ready to go it will be reduced down to the final approach.

-Dale

### CODE REVIEW

File: libmscore/scoreElement.h [hunk 1]
```diff
@@ -171,6 +171,9 @@ struct ElementName {
 
 class ScoreElement {
       Score* _score;
+#ifdef SCRIPT_INTERFACE
+      QObject* _plugInWrapper;
```
Comment (dmitrio95) [inline libmscore/scoreElement.h]: You can define a default value for this variable just here:
`QObject* _plugInWrapper = nullptr;`
Then there will be no need in having two different constructors depending on `SCRIPT_INTERFACE` definition and in initializing this value explicitly in copy constructor.

Comment (DLLarson) [reply libmscore/scoreElement.h]: Thanks! Much cleaner now.

-Dale

**Comment by Jojo-Schmitz** [comment]
rebase neeeded

**Comment by vpereverzev** [comment]
**Automatically archived due to long inactivity. Could be re-opened later once an author will come back**

**Pull request closed without merging**